nikola/data/__init__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
Adding `__init__.py` files is unacceptable. Please try to find a solution based on what was discussed before that does not involve `__init__.py` files.

Reply:
If you can’t get importlib to work correctly with 3.8, you can add a dependency. Make sure it’s used and installed only on 3.8 (make the import and requirement conditional).